### DISCUSSION TITLE
重求一个活跃橙

### DISCUSSION THREAD

**Discussion by Griseo-bh3rd** [Meloong-Git/PCL#3262] [讨论]
如题，重装完电脑活跃橙没了，重新求一个

识别码 E2CF-7AD6-E742-838D

#1066 bug issue

谢谢龙猫了

**Comment by allMagicNB** [comment]
> 如题，重装完电脑活跃橙没了，重新求一个
> 
> 识别码 E2CF-7AD6-E742-838D
> 
> #1066 bug issue
> 
> 谢谢龙猫了
> 

直接在原帖以下再次要求，不用开Discussion

**Comment by Light-Beacon** [comment]
你的解锁码：`cJ2SL+XLqyXf0PF7hHdiamYuxlZkYbqx`

**Reply by allMagicNB** [reply]
~~他谢谢的是龙猫~~
